packages/rest/test/routes.test.ts

```diff
@@ -705,42 +695,16 @@ describe("rest: routes", () => {
       namespace Things {
         @action
         @actionSeparator(":")
-        @put op customAction1(
-          @segment("things")
-          @path thingId: string
-        ): string;
-
-        #suppress "deprecated"
-        @action
-        @segmentSeparator(":")
-        @put op customAction2(
+        @put op customAction(
           @segment("things")
           @path thingId: string
         ): string;
-
-        @get op getAccount(
```

Comment:
same question

Reply:
This one is removed because you can't have "Microsoft.Account/" as an action separator, so the logic in the cadl-azure PR was updated accordingly to not rely on this iffy use of segment separator.